release/ray_release/alerts/tune_tests.py

```diff
@@ -44,7 +44,7 @@ def handle_result(
         target_time = 900 if not was_smoke_test else 400
     elif test_name == "result_throughput_cluster":
         target_terminated = 1000
-        target_time = 160
+        target_time = 130
```

Comment:
I changed a few more `130`'s than this one https://github.com/ray-project/ray/pull/32062/files

Reply:
Thanks, updated! Running both tests now